receptai.api/Controllers/CommentVoteController.cs

```diff
@@ -64,7 +64,7 @@ public async Task<IActionResult> GetByCommentId([FromRoute] int commentId)
 
         var commentVoteModel = commentVoteDto.ToCommentVoteFromCreateDto();
         commentVoteModel.UserId = User.GetId();
-        await _commentVoteRepository.CreateAsync(commentVoteModel);
+        await _commentVoteRepository.CreateAsync(commentVoteModel, User.GetId());
```

Comment:
comment vote model already has UserId which is filled above this file, so second argument is redundant (same for recipeVoteModel)

Reply:
good catch, did not spot this myself somehow :)